Orange/widgets/visualize/owsilhouetteplot.py

```diff
@@ -507,7 +504,10 @@ def setRowNames(self, names):
             item = layout.itemAt(i + 1, 3)
 
             if grp.rownames is not None:
-                item.setItems(grp.rownames)
+                rownames = [rowname[0:12] + "…" if
```

Comment:
However, this is not the proper way to do it since it disregards the actual widths of characters. See if you can truncate the text with http://doc.qt.io/qt-4.8/qfontmetrics.html#elidedText instead. For this, you will have to decide for (instead of determining) the sensible maximal label width in advance.